### ISSUE TITLE
Hover off not working after editing

### ISSUE THREAD

**Issue by jonbondani** [giorgosart/react-easy-edit#72]
After editing a value, if you click on cancel, the item's value remains italic.

Using the  provided sandbox you can see it in action:
https://codesandbox.io/s/react-easy-edit-sandbox-2y97j

Digging into the code it seems that after both _onSave and _onCancel function you have to add the following:
```
 o(d(s), "_onSave", function() {
        var e = s.props,
            t = e.onSave,
            a = e.onValidate,
            n = s.state.tempValue;
        a(n) ? s.setState({
            editing: !1,
            value: n,
            isValid: !0,
            ** hover: !1 **
        }, function() {
            return t(s.state.value)
        }) : s.setState({
            isValid: !1
        })
    }), o(d(s), "_onCancel", function() {
            var e = s.props.onCancel,
                t = s.state.value;
            s.setState({
                editing: !1,
                tempValue: t,
                ** hover: !1 **
            }, function() {
                return e()
            })
        }
```

**Comment by giorgosart** [comment]
Hey @jonbondani  thank you for logging a ticket, let me review it and i will get back to you as soon as possible

**Comment by giorgosart** [comment]
Hey @jonbondani i'll fix this today and get it out probably sometime tonight

**Comment by giorgosart** [comment]
Hey @jonbondani, issue is now fixed as part of [1.8.1](https://github.com/giorgosart/react-easy-edit/releases/tag/1.8.1)
